clients/cloud/ruby/README.md

```diff
@@ -9,7 +9,7 @@ Produce messages to and consume messages from [Confluent Cloud](https://www.conf
 * Access to a [Confluent Cloud](https://www.confluent.io/confluent-cloud/) cluster
 * Local file with configuration parameters to connect to your Confluent Cloud instance ([how do I find those?](https://docs.confluent.io/current/cloud/using/config-client.html#librdkafka-based-c-clients)). Format the file as follows:
     ```bash
-    $ cat ~/.ccloud/librdkafka.config
+    $ cat ~/.ccloud/example.config
```

Comment:
@ybyzek Maybe we should use this filename in all of the examples? The files are identical anyway, and the example filename avoids it getting confused with any other config files in ccloud/

Reply:
I also think so. I pushed a commit to do that - cc @ybyzek